### PR TITLE
Added newlines to generated cask json.

### PR DESCRIPTION
**Pull request by cmbernard333**

### CODE REVIEW

File: script/generate-cask.rb [hunk 1]
```diff
@@ -7,5 +7,6 @@
 Cask::Cask.each do |c|
   json_filename = "#{c.token}.json"
   IO.write("_data/cask/#{json_filename}", JSON.pretty_generate(c.to_h))
```
Comment (MikeMcQuaid) [inline script/generate-cask.rb]: Could this use `IO.puts` instead? If not, how about doing `#{JSON.pretty_generate(c.to_h)}\n`?

Comment (cmbernard333) [reply script/generate-cask.rb]: Whatever makes the most sense. I was just using my limited ruby knowledge here. XD

File: script/generate-cask.rb [hunk 2]
```diff
@@ -6,6 +6,6 @@
 
 Cask::Cask.each do |c|
   json_filename = "#{c.token}.json"
-  IO.write("_data/cask/#{json_filename}", JSON.pretty_generate(c.to_h))
+  IO.write("_data/cask/#{json_filename}", {JSON.pretty_generate(c.to_h)}\n)
```
Comment (reitermarkus) [inline script/generate-cask.rb]: This isn't valid syntax, should be

```ruby
"#{JSON.pretty_generate(c.to_h)}\n"
```

**Comment by MikeMcQuaid** [comment]
Thanks @cmbernard333!

**Pull request merged**